.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
```

Comment:
We should add a reference to the main repo for bug reports and feature requests on Sequelize itself

Reply:
IMO most of these links could go away and instead have these 3 choices:
- Documentation issues (template in this repo)
- Help & Questions (it's related to docs so it makes sense to add it just in case)
- For everything else: https://github.com/sequelize/sequelize/issues/new/choose